Makefile

```diff
@@ -86,6 +86,11 @@ propose-jwt-ms-validation-policy: ## 🚀 Propose the AAD as idp
 	@./scripts/generate_jwt_proposal_payload.sh  --proposal-file "${KMS_WORKSPACE}"/proposals/set_jwt_ms_validation_policy_proposal.json
 	@CCF_PLATFORM=${CCF_PLATFORM} ./scripts/submit_proposal.sh --network-url "${KMS_URL}" --proposal-file "${KMS_WORKSPACE}"/proposals/set_jwt_ms_validation_policy_proposal.json --certificate_dir "${KEYS_DIR}" --member-count ${MEMBER_COUNT}
 
+# Propose a new settings policy
+propose-settings-policy: ## 🚀 Deploy the settings policy
+	@echo -e "\e[34m$@\e[0m" || true
+	@CCF_PLATFORM=${CCF_PLATFORM} ./scripts/submit_proposal.sh --network-url "${KMS_URL}" --proposal-file ./governance/policies/settings-policy.json --certificate_dir "${KEYS_DIR}" --member-count ${MEMBER_COUNT}
```

Comment:
Can you add description in README about `settings-policy.json` and how it's intended to be used please?

Reply:
done